docs/crud.md

```diff
@@ -0,0 +1,103 @@
+# Access a document
+
+#### Request
+```http
+GET /data/:type/:id
+```
+```http
+GET /data/types.cozy.io/events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee
+```
+
+#### Response OK
+```http
+HTTP/1.1 200 OK
+Date: Mon, 27 Sept 2016 12:28:53 GMT
+Content-Length: ...
+Content-Type: application/json
+Etag: "3-6494e0ac6494e0ac"
+```
+```json
+{
+    "_id": "types.cozy.io/events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee",
+    "_rev": "3-6494e0ac6494e0ac",
+    "_type": "types.cozy.io/events",
+    "startdate": "20160823T150000Z",
+    "enddate": "20160923T160000Z",
+    "summary": "A long month",
+    "description": "I could go on and on and on ....",
+}
+```
+
+### Response Error
+```http
+HTTP/1.1 404 Not Found
+Content-Length: ...
+Content-Type: application/json
+```
+```json
+{
+  "status": 404,
+  "error": "not_found",
+  "reason": "deleted",
+  "title": "Event deleted",
+  "details": "Event 6494e0ac-dfcb-11e5-88c1-472e84a9cbee was deleted",
+  "links": {"about": "https://cozy.github.io/cozy-stack/errors/deleted.md"}
+}
+```
+
+### possible errors :
+- 403 unauthenticated
+- 401 unauthorized
+- 404 not_found
+  - reason: missing
+  - reason: deleted
+- 500 unkown
+
+--------------------------------------------------------------------------------
+
+# Create a document
+
+### Request
+```http
+POST /data/:type/
```

Comment:
I must admit I am not sure about this part as it is indeed a duplicate information.

Having it in the URL allow us to access it without parsing the whole body, eventually ignoring the body if we know from the type this request wont be accepted (permission/protection against memory bomb) or starting some stream-parsing-analysis without keeping the whole body in memory if we know we will need it (images).
 
On the other hand, having it in the JSON body itself might prove helpful for consumption from client-side frameworks, this is more of a gut feeling, so I am not sure we wil actually need it in the body.